tiatoolbox/wsicore/wsireader.py

```diff
@@ -1682,7 +1674,7 @@ def read_bounds(
             read_kwargs=kwargs,
         )
 
-        if location_is_at_requested:
+        if coords_space == 'resolution':
```

Comment:
Very minor thing, but here and in the function signature you use single quotes. I believe black should have changed these to double quotes? Could you double-check this file has been run through black to format it?